### PR TITLE
fixes browser crash when session<->analysis

### PR DESCRIPTION
**Pull request by TS-Stefan**
fixes timeline scaling when audio duration is very long

### CODE REVIEW

File: apps/koala-frontend/src/app/features/sessions/pages/session/session.page.ts [hunk 1]
```diff
@@ -185,6 +185,7 @@ export class SessionPage implements OnInit, OnDestroy {
     this.sessionUpdatedSubscription?.unsubscribe();
     this.toolbarUpdatedSubscription?.unsubscribe();
     this.timerSubscription?.unsubscribe();
+    this.mediaControlService.destroy();
```
Comment (TS-Daniel16) [inline apps/koala-frontend/src/app/features/sessions/pages/session/session.page.ts:188]: Why is this necessary? Not sure if we should interfere with angular managed / injected services

Comment (TS-Stefan) [reply apps/koala-frontend/src/app/features/sessions/pages/session/session.page.ts:188]: It destroys the wavesurfer not the actual angular service

Comment (TS-Stefan) [reply apps/koala-frontend/src/app/features/sessions/pages/session/session.page.ts:188]: Without the whole browser crashes when u switch between session and analysis. Took me quite some time to debug this!

Comment (TS-Stefan) [reply apps/koala-frontend/src/app/features/sessions/pages/session/session.page.ts:188]: https://github.com/KoALa-MHF/koala-app/blob/main/apps/koala-frontend/src/app/features/sessions/services/media-control.service.ts#L259

**Review by TS-Tino16 — APPROVED** [review]
